IronQuotes/src/main/java/com/theironyard/Command/TagCommand.java

```diff
@@ -0,0 +1,19 @@
+package com.theironyard.Command;
+
+/**
+ * Created by Nigel on 8/1/16.
+ */
+public class TagCommand {
+    private String value;
+
+    public TagCommand() {
```

Comment:
If you don't have a custom constructor then you don't have to add the default one as it will inherit from `Object`